### PR TITLE
Upgrade to jammy

### PR DESCRIPTION
**Pull request by marceloneppel**
# Issue
Jira issue: https://warthogs.atlassian.net/browse/DPE-1413
The charm is still on Focal.

# Solution
Upgrade all the bases and all the deployments to use Jammy.

# Context
The Landscape bundle test was disabled due to incompatibilities. Ticket to manage those incompatibilities: https://warthogs.atlassian.net/browse/DPE-1414

A lot of changes in the tests related to the database app name constant are due to removing multiple series in the tests.

# Testing
Used the existing tests to confirm that the charm works on Jammy.

# Release Notes
Upgrade charm to Jammy.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
           - charm-integration
           - database-relation-integration
           - db-relation-integration
-          - db-admin-relation-integration
+#          - db-admin-relation-integration
```
Comment (dragomirp) [inline .github/workflows/ci.yaml:51]: Might be better to mark it as `unstable`

**Review by dragomirp — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (marceloneppel) [reply .github/workflows/ci.yaml:51]: I tried with it before, but the test failed. I think it's because there is only one test in that file.

Comment (taurus-forever) [reply .github/workflows/ci.yaml:57]: @carlcsaposs-canonical can you please check this case and improve unstable concept... commenting tests makes them invisible. 

@marceloneppel please create a bugreport ticket to re-enable the test. So we can schedule it in a pulse.

File: tests/integration/new_relations/test_new_relations.py [hunk 2]
```diff
@@ -42,20 +42,21 @@ async def test_deploy_charms(ops_test: OpsTest, application_charm, database_char
...
                 application_name=APPLICATION_APP_NAME,
                 num_units=2,
+                series=CHARM_SERIES,
             ),
             ops_test.model.deploy(
                 database_charm,
                 resources={"patroni": "patroni.tar.gz"},
                 application_name=DATABASE_APP_NAME,
                 num_units=1,
-                trust=True,
```
Comment (taurus-forever) [inline tests/integration/new_relations/test_new_relations.py:51]: JFMI, why trust is no longer necessary?

**Review by taurus-forever — APPROVED** [review]
Thank you!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (marceloneppel) [reply .github/workflows/ci.yaml:57]: Created: https://warthogs.atlassian.net/browse/DPE-1414

File: tests/integration/new_relations/test_new_relations.py (hunk 2 above)
Comment (marceloneppel) [reply tests/integration/new_relations/test_new_relations.py:51]: It was not needed before. Probably it was incorrectly copied from the k8s charm tests.

**Pull request merged**